
## Our dataset

Mass Spectrometry-Based Proteomic Profiling of Thrombotic Material Obtained by Endovascular Thrombectomy in Patients with Ischemic Stroke.[pubmed](https://www.ncbi.nlm.nih.gov/pubmed?term=29414888)

> MS/MS data were obtained using AnalystTF 1.5.2 (Sciex, Washington, WA, USA), and spectra
files were processed through Protein Pilot TM Software 5.0 (Sciex, Washington, WA, USA) using
ParagonTM Algorithm 5.0 [57] and searched against the concatenated target-decoy UniProt human
database.

Data: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD007666

## Additional datasets

Machine Learning Analysis of the Cerebrovascular Thrombi Proteome in Human Ischemic Stroke: An Exploratory Study [pubmed](https://pubmed.ncbi.nlm.nih.gov/33240201/)

> The resulting peptide samples were analyzed online using Q- Exactive HF mass spectrometer coupled with an Ultimate 3000 RSLC (Thermo Fisher Scientific) fitted with a stainless-steel emitter (Thermo Fisher Scientific). Spectra were acquired with Xcalibur software (v4.1 Thermo Fisher Scientific). MS/MS analyses were performed in a data-dependent mode with standard settings. MS data analysis was performed using the MaxQuant software with default settings (v1.5.5.1)

Data: http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD020398

Potential Biomarkers of Acute Ischemic Stroke Etiology Revealed by Mass Spectrometry-Based Proteomic Characterization of Formalin-Fixed Paraffin-Embedded Blood Clots [pubmed](https://pubmed.ncbi.nlm.nih.gov/35518205/)

> The thrombus proteome was analyzed by label-free quantitative liquid chromatography-
tandem mass spectrometry (LC-MS/MS). MaxQuant v1.5.2.8 and Perseus v.1.6.15.0
were used for bioinformatics analysis.

Data: 
  - http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD031477
  - https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=40990ab31c384c87b5ce98286af51d6a
  
Integrative Multi-omics Analysis to Characterize Human Brain Ischemia [pubmed](https://pubmed.ncbi.nlm.nih.gov/33939164/)

>  Proteomics and transcriptomics discovery studies were performed in human brain samples from six deceased stroke patients, comparing the infarct core with the corresponding contralateral brain region, unveiling 128 proteins and 2716 genes significantly dysregulated after stroke. Integrative bioinformatics analyses joining both datasets exposed canonical pathways altered in the ischemic area, highlighting the most influential molecules. Among the molecules with the highest fold-change, 28 genes and 9 proteins were selected to be validated in five independent human brain samples using orthogonal techniques. Our results were confirmed for NCDN, RAB3C, ST4A1, DNM1L, A1AG1, A1AT, JAM3, VTDB, ANXA1, ANXA2, and IL8. 

Data: https://www.ebi.ac.uk/pride/archive/projects/PXD022850
